tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "skipLibCheck": true,
-    "baseUrl": "./arches/app/src/",
+    "baseUrl": "**/src/",
```

Comment:
With this change, tsc no longer finds my type declarations like:

```ts
import type {
    ControlledList,
    ControlledListItem,
    NewItem,
} from "@/types/ControlledListManager";
```

giving instead:
```js
arches/app/src/components/ControlledListManager/utils.ts:7:8 - error TS2307: Cannot find module '@/types/ControlledListManager' or its corresponding type declarations.

7 } from "@/types/ControlledListManager";
```

I'm going to revert this change (and the related change in the templates and the updateproject.py) so that we can pull these changes in and unblock 8.0 work, but I would gladly re-review these changes in another a PR if you would like to discuss it futher.